### PR TITLE
Staff create and export bill

### PR DESCRIPTION
**Pull request by phucynwa**

### CODE REVIEW

**Comment by phucynwa** [comment]
https://dev.framgia.com/issues/184354

**Comment by phucynwa** [comment]
ready

**Comment by phucynwa** [comment]
ready

File: app/controllers/admin/bills_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  private
+
+  def bill_params
+    params.require(:bill).permit :code, :booking_id
+  end
+
+  def load_bill
+    @bill = Bill.find_by id: params[:id]
```
Comment (huongnguyenmta) [inline app/controllers/admin/bills_controller.rb]: check `nil`

**Comment by huongnguyenmta** [comment]
commented

**Comment by phucynwa** [comment]
ready

**Comment by phucynwa** [comment]
ready

**Pull request merged**